.github/workflows/macos.yml

```diff
@@ -70,7 +70,14 @@ jobs:
         working-directory: _build
       # Try brew install of head: If error message below appears, build and test passed successfully
       - name: brew install test
-        run: brew install --HEAD --formula -s oqsprovider.rb 2>&1 | grep "Empty installation"
+        # try this only if brew'd liboqs knows about ML-KEM:
+        run: |
+          bash -c 'brew install liboqs && FIND_KEM=`brew info liboqs | grep files | awk "{print $1}"`
+            if [[ `echo $FIND_MLKEM | grep ML_KEM` ]]; then
```

Comment:
The variable name on line 76 is `FIND_MLKEM` but on line 75 it's `FIND_KEM`, is that a mistake? Same in other files below.

Reply:
Thanks very much! Stupid mistake :-(